neptune/new/exceptions.py

```diff
@@ -45,6 +45,28 @@ class MetadataInconsistency(NeptuneException):
     pass
 
 
+class MissingAttributeException(NeptuneException):
+    """Raised when get-like action is called on `Handler`, instead of on `Attribute`."""
+
+    def __init__(self):
+        message = """
+{h1}
+----MissingAttributeException---------------------------------------------------
+{end}
+Neptune Client Library can't access given `Attribute`.
+
+Probably `Attribute` wasn't synced with backend API yet.
+You can sync it by either calling:
+* `.wait()`
+* `.sync()`
+* or creating an attribute with option `wait=True`.
```

Comment:
Same here. Wait will work in cases when client knows about attribute but server does not. Here we are talking about opposite cases.

Reply:
ok